styx-service-common/src/test/java/com/spotify/styx/storage/DatastoreStorageTest.java

```diff
@@ -222,10 +225,35 @@
 
   private static final WorkflowId WORKFLOW_ID = WorkflowId.create("dockerComp", "dockerEndpoint");
 
+  private static final String DOCKER_IMAGE = "gcr.io/foo/bar";
   private static final WorkflowConfiguration WORKFLOW_CONFIGURATION =
       WorkflowConfiguration.builder()
           .id(WORKFLOW_ID.id())
           .schedule(DAYS)
+          .dockerImage(DOCKER_IMAGE)
+          .build();
+  private static final WorkflowConfiguration DOCKER_EXEC_WORKFLOW_CONFIGURATION =
```

Comment:
Is there a reason we don't use the TestData.java declaration of these variables?

Reply:
No really, I will move code overthere